src/Squirrel.CommandLine/OSX/Commands.cs

```diff
@@ -55,19 +48,19 @@ private static void Pack(PackOptions options)
                 Log.Info("Pack directory is not a bundle. Will generate new '.app' bundle from a directory of application files.");
 
                 if (options.icon == null || !File.Exists(options.icon))
-                    throw new OptionValidationException("--icon is required when generating a new app bundle.");
+                    throw new ArgumentException("--icon is required when generating a new app bundle.");
```

Comment:
Is there a way to get the command alias from System.CommandLine so that "--icon" is not hard-coded here?

Reply:
There are a couple of ways to do this. The first is to simply grab the alias for the option (Generally grabbing the longest one):
`new PackCommand().Icon.Aliases.OrderByDescending(x => x.Length).First();`
There is also a `Name` property on the Options, but this property omits the prefix (`-` or `--`).

IMO a better solution is to use whichever alias the caller specified when invoking the CLI, as this feel more natural for an error message. However to know which alias was specified requires inspecting the `ParseResult`. This means prompting the user with the error while the command line is parsed rather than doing the validation after the command is invoked. You can see several examples of this in the custom validation that I put inside of the `SystemCommandLineExtensions`. 

I suspect many of these failure cases could be tested for during the parse options (and in fact the icon one should already be covered). So this validation checking could be removed. I had just done my best to avoid touching it because the PR was already getting large.